packages/paste-website/src/pages/tokens/index.mdx

```diff
@@ -11,7 +11,7 @@ import {TokensList} from '../../components/tokens-list';
 
 # Design Tokens
 
-Design Tokens are named entities that store visual design attributes. Tokens are named for their use case, rather than their value. For example `$color-background-error` is named to indicate the appropriate color for an error, as opposed to `$color-background-red` which is less specific and could be used in a number of ways depending on cultural context and design language. This makes their intended use clear and intentional, and allows us to develop a scalable and consistent visual language.
+Design Tokens are named entities that store visual design attributes. Tokens are named for their use case, rather than their value. For example `$color-background-error` is named to indicate the appropriate background color for an error, as opposed to `$color-background-red` which is less specific and could be used in a number of ways depending on cultural context and design language. This makes their intended use clear and intentional, and allows us to develop a scalable and consistent visual language.
```

Comment:
Suggested rewrite for first sentence: Design Tokens store design-related attributes, such as colors, spacing, and animations.